packages/macros/src/ember-addon-main.ts

```diff
@@ -25,8 +25,21 @@ export = {
     let babelOptions = (parentOptions.babel = parentOptions.babel || {});
     let babelPlugins = (babelOptions.plugins = babelOptions.plugins || []);
     babelPlugins.unshift(MacrosConfig.shared().babelPluginConfig(source));
+
+    // TODO: LOL MEGA HACK
```

Comment:
We should discuss this

Reply:
yes we must

Reply:
If we decide this is the correct timing, I don't mind adding the appropriate hook to ember-cli proper, and using this as a temp polyfil.

cc @ef4